manifests/a/Asus/GPUTweakIII/1.5.8.0/Asus.GPUTweakIII.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=AUSU.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Asus.GPUTweakIII
+PackageVersion: 1.5.8.0
+InstallerType: zip
```

```suggestion
InstallerType: zip
AppsAndFeaturesEntries:
  - DisplayName: GPU Tweak III Install
```
I'm not sure if this is the right way to go about it, so I'm putting it here to get an opinion. Should this be put in PackageName field or here as an ARP entry? I would want user to see `GPU Tweak III` in winget search / show / install, but I also don't want to cause issues with winget upgrade logic. We do have an exact match with ProductCode (and WinGet's fuzzy matching for PackageName shouldn't be causing any conflicts here?), so I'd say we don't need the ARP entry?

---

The installer adds two ARP entries - One that is just `GPU Tweak III` and one that is `GPU Tweak III Install`. The product code will cause it to match on the one without `Install`

```
--> Comparing ARP Entries

DisplayName                                                    DisplayVersion Publisher             ProductCode
-----------                                                    -------------- ---------             -----------
Microsoft Visual C++ 2017 x64 Minimum Runtime - 14.11.25325    14.11.25325    Microsoft Corporation {B0037450-526D-3448-A370-CACBD87769A0}
Microsoft Visual C++ 2017 x64 Additional Runtime - 14.11.25325 14.11.25325    Microsoft Corporation {B13B3E11-1555-353F-A63A-8933EE104FBD}
Microsoft Visual C++ 2017 x86 Minimum Runtime - 14.11.25325    14.11.25325    Microsoft Corporation {029DA848-1A80-34D3-BFC1-A6447BFC8E7F}
GPU Tweak III Install                                          1.5.8.0        ASUS                  {0d3769a0-fd52-4693-8744-4671f017952f}
Microsoft Visual C++ 2017 Redistributable (x86) - 14.11.25325  14.11.25325.0  Microsoft Corporation {404c9c27-8377-4fd1-b607-7ca635db4e49}
Microsoft Visual C++ 2017 x86 Additional Runtime - 14.11.25325 14.11.25325    Microsoft Corporation {568CD07E-0824-3EEB-AEC1-8FD51F3C85CF}
Microsoft Visual C++ 2017 Redistributable (x64) - 14.11.25325  14.11.25325.0  Microsoft Corporation {6c6356fe-cbfa-4944-9bed-a9e99f45cb7a}
GPU Tweak III                                                  1.5.8.0        ASUS                  {8753C623-2580-4C38-A5C6-229FF2A940A3}
```

---

Oh, looks like I missed the correct ARP entry 🤦‍♂️